root/components/FormRowCheckbox.js

```diff
@@ -32,6 +33,9 @@ const FormRowCheckbox = ({field, label, onChange}: Props) => (
       {label}
     </label>
     <FieldErrors field={field} />
+    <div className="form-help">
```

Comment:
This should probably output `null` instead of an empty div if `help` isn't given.

Also, a nice accessibility tweak we can add: ``id={`field-help-${field.id}`}`` to the div here, and ``aria-describedby={help ? `field-help-${field.id}` : null}`` to the input above.

Reply:
Oooh. I like that :) We should add more of those.